src/main/java/bio/terra/cli/command/user/sshkey/Add.java

```diff
@@ -25,15 +25,16 @@ public class Add extends BaseCommand {
   @Override
   protected void execute() {
     ExternalCredentialsManagerService ecmService = ExternalCredentialsManagerService.fromContext();
-    var sshKeyPair = ecmService.getSshKeyPair(SshKeyPairType.GITHUB, /*includePrivateKey=*/ true);
+    SshKeyPair sshKeyPair =
+        ecmService.getSshKeyPair(SshKeyPairType.GITHUB, /*includePrivateKey=*/ true);
     saveKeyFileAndSshAdd(sshKeyPair);
     formatOption.printReturnValue(UFSshKeyPair.createUFSshKey(sshKeyPair), UFSshKeyPair::print);
   }
 
   protected static void saveKeyFileAndSshAdd(SshKeyPair sshKeyPair) {
-    var dir = System.getProperty("user.home") + "/.ssh/";
```

Comment:
i still think we don't need to specify they are string, it's pretty clear

Reply:
reverting string change here. Not against string being inferred